src/backend/aspen/api/views/users.py

```diff
@@ -1,15 +1,19 @@
-from fastapi import APIRouter, HTTPException
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
 from starlette.requests import Request
 
+from aspen.api.deps import get_db
 from aspen.api.schemas.users import User as userschema
 from aspen.database.models.usergroup import User
 
 router = APIRouter()
 
 
 @router.get("/me", response_model=userschema)
-async def get_current_user(request: Request) -> userschema:
-    instance = await User.get_by_id(request.state.auth_user.id)
+async def get_current_user(
+    request: Request, db: AsyncSession = Depends(get_db)
+) -> userschema:
+    instance = await User.get_by_id(db, request.state.auth_user.id)
```

Comment:
Isn't `request.state.auth_user` already the whole User object? Do we need to get it by its ID again, don't we already have it?

Reply:
Yeah, though this endpoint is intended as an example of how we'd fetch <some resource> by an ID in the path.